### DISCUSSION TITLE
Dark mode on the prototyping kit

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1872] [Q&A] [answered]
Hi, 

Would it be possible to update the Paste prototyping kit so that it works for dark mode? Its just the masthead that is missing some borders from what I can see so far.

![image](https://user-images.githubusercontent.com/72031986/133653813-d5f78e51-b099-441c-b636-3e2789bc6b18.png)

**Comment by SiTaggart** [comment, marked as answer]
Hi @PoojMenon, resources are a little tight for us right now, we might not get to this until towards the end of the next release in Novemeber.

We do however, accept [Pull Requests](https://github.com/twilio-labs/paste-prototype-kit/) if you are feeling up for it?

**Reply by andioneto** [reply]
@PoojMenon Is there anything else that we can help you with, with regard to this question?

**Reply by PoojMenon** [reply]
I'd be happy to contribute - its a fairly minor change. I'll work on this some time this week or the next.

**Reply by SiTaggart** [reply]
So pumped for this. 🎉

**Reply by PoojMenon** [reply]
My very first [PR](https://github.com/twilio-labs/paste-prototype-kit/pull/56) created - hopefully didn't mess that up.
